core/txpool/validation.go

```diff
@@ -69,6 +69,9 @@ func ValidateTransaction(tx *types.Transaction, head *types.Header, signer types
 	if tx.Type() == types.DepositTxType {
 		return core.ErrTxTypeNotSupported
 	}
+	if opts.Config.IsOptimism() && tx.Type() == types.BlobTxType {
```

Comment:
Are we gonna test that we reject l2 blob transactions with the op-e2e test?

Reply:
yes, tested in https://github.com/ethereum-optimism/optimism/pull/8707/commits/7cc3996276e4f987a248312d130d4fcfa5ea3454